cmd/docker/main.go

```diff
@@ -218,6 +218,7 @@ func startServerWithOptions(cli *CliOptions) *apiserver.Server {
 			// server requires and verifies client's certificate
 			tlsConfig.ClientAuth = tls.RequireAndVerifyClientCert
 			tlsConfig.ClientCAs = loadCAPool()
+			tlsConfig.InsecureSkipVerify = true
```

Comment:
Shouldn't this be `false` here?  If not, can you add a comment where/when it'll get set to false?